Detectors/TPC/qc/include/TPCQC/Helpers.h

```diff
@@ -50,6 +50,14 @@ void setStyleHistogram2D(TH2& histo);
 /// Set nice style for vector of 2D histograms
 void setStyleHistogram2D(std::vector<TH2F>& histos);
 
+/// set nice style for 1D histograms ptr
+void setStyleHistogram1D(TH1* histo);
+
+/// set nice style for 2D histograms ptr
+void setStyleHistogram2D(TH2* histo);
```

Comment:
See also comments below
```suggestion
/// set nice style for histograms
void setStyleHistogram(TH1& histo);
```